chromadb/test/test_chroma.py

```diff
@@ -65,3 +65,28 @@ def test_fastapi(self, mock: Mock) -> None:
             )
         )
         assert mock.called
+
```

Comment:
@HammadB does this test make sense?

Reply:
This makes sense in terms of the fact that it tests headers being passed into the client. However it doesn't test that the headers are sent out in the requests.

I can see how that could be complicated, so as long as we manually verify it - I could be OK with that.

Reply:
verified I can catch this serverside manually. here are the screenshots

<img width="859" alt="Screenshot 2023-07-13 at 1 45 20 PM" src="https://github.com/chroma-core/chroma/assets/891664/c0c3ccfa-fc8d-486b-a664-7b1139f2dd39">
<img width="705" alt="Screenshot 2023-07-13 at 1 45 13 PM" src="https://github.com/chroma-core/chroma/assets/891664/4f98407c-cff0-47cf-b279-6cd4b8e413e9">